Math.md

```diff
@@ -0,0 +1,96 @@
+##Float and Double
+You've already learned about the String and Int types; Strings are values like "hello" or "eat your vegetables", and Ints are whole number values like 34 and 9561.  
+
+Strings may form the backbone of displaying instructions and values to the user, but numbers (and especially mathematical operations on those numbers) are necessary for actually drawing that text to the screen, as well as pretty much every thing the operating system and your applications do, from computing how many days are left in a subscripton to centering a button in window.  
+
+Another kind of number are Float and Double, which represent decimal values such as 0.5 or 3.14159.  The difference between Float and Double is magnitude and precision: Floats are represented in 32 bits and Doubles are represented in 64 bits, so Doubles have the potential for greater magnitude and precision.
+
+You can recognize Floats and Doubles by value because they will have a decimal point; the Swift compiler will interpret any number with a decimal point as a Double unless you specify that it should be recognized as a Float.  Any number without a decimal point will be interpreted by the compiler as an Int.
+
+##Mathematical Operations
+The four primary operators in math are addition (+), subtraction (-), multiplication (*), and division (/); the fifth operator is remainder (%).  For example:
+
+```Swift
+5 / 3 = 1
+5 % 3 = 2
+````
+ 
+ Similarly, you can assign Int and Float values to variables and perform math operations.  For example:
+
+ ```Swift
+ let a = 4
+ let b = 12
+ let n = b / a   (3)
+ let r = b % a   (0)
+ let product = a * b   (48)
+ let sum = a + b   (16)
+ let difference = b - a   (8)
+ ````
+
+ and using Floats...  (because there's a decimal the compiler knows it's a Float)
+
+````Swift
+ let x = 5.5
+ let y = 10.0
+ let n = y / x  (1.8181...)
+ let r = y % x   (4.5)
+ let proudct = x * y   (55)
+ let sum = x + y  (15.5)
+ let difference = y - x  (4.5)
+ ````
```

Comment:
Similar to my above comment with regards to the comment lines // instead of ( ), but also.. I think we could be really explicit here so that way we drill into their brains how they can describe these various constants being created here like so:

```swift
let x = 5.5
// x is a constant of type Double with a value of 5.5

let y = 10.0
// y is a constant of type Double with a value of 10.0

let n = y / x
// n is a constant of type Double with a value of 1.818181818181818

let r = y % x
// r is a constant of type Double with a value of 4.5

let prouduct = x * y
// product is a constant of type Double with a value of 55

let sum = x + y
// sum is a constant of type Double with a value of 15.5

let difference = y - x
// difference is a constant of type Double with a value of 4.5
```